### DISCUSSION TITLE
Diagrams.net Lucid Charts Import Tool

### DISCUSSION THREAD

**Discussion by jerremywi** [jgraph/drawio#2753] [Q&A]
I have been trying to use the following tool to import diagrams from Lucid charts into Diagrams.net
https://app.diagrams.net/tools/lucid/import.html

When I export my Lucid Charts diagrams it saves them into a zip file with individual .drawio files

When I save these files into my google drive and try to open them using diagrams.net I get the following error.

Error Loading File. Not a diagram file

![image](https://user-images.githubusercontent.com/103551743/163095177-848ffbdc-03e9-46c4-a819-ceb345e76d50.png)

Am I doing something wrong or is it broken?

**Comment by davidjgraph** [comment]
What does the start of the file that you tried to open look like in a text editor?

**Reply by jerremywi** [reply]
Looking at the files sizes they definitely don't appear to be full files.

This is a sample from one of the files. This file is fact and opinion.drawio

![image](https://user-images.githubusercontent.com/103551743/163334339-189b5551-14e7-42d4-88a3-e5d9ab89da43.png)

**Comment by jerremywi** [comment]
![image](https://user-images.githubusercontent.com/103551743/163334028-ef397a94-feed-45c3-a010-8cf3d7f58307.png)

**Reply by m-mohamedin** [reply]
I've tried importing from Lucid then upload to Google Drive and it works. Your file also looks correct. 

Can you please try to load the file directly in draw.io editor (via drag and drop or File -> Import). Does it work?

If that didn't work, is it possible to send us the file
